mmv1/provider/template_data.go

```diff
@@ -105,13 +104,6 @@ func (td *TemplateData) GenerateFile(filePath, templatePath string, resource api
 		sourceByte = bytes.Replace(sourceByte, []byte("github.com/hashicorp/terraform-provider-google/google"), []byte(td.TerraformProviderModule+"/"+td.TerraformResourceDirectory), -1)
 	}
 
```

Comment:
Why delete this part?

Reply:
The file generation was failing due to the gofmt breaking (which is expected due to the file still incomplete and having spots of broken syntax) so I had to remove it for working down the file -- I'll restore it for the PRs or could swap it to a comment while these are worked through